### PR TITLE
feat(csms): add a data source to query the CSMS secret version

### LINKED ISSUE #1897: I want to add a data source for querying the plaintext of the CSMS secrets
I want to add a data source for querying the plaintext of the CSMS secrets.
The data source name: `huaweicloud_dew_secret_version`

### PR DESCRIPTION
**Pull request by chengxiangdong**
**What this PR does / why we need it**:

Add a data source to query the CSMS secret version.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #1897

**Special notes for your reviewer**:

Some functions in `huaweicloud_csms_secret` is called, pls merge #1889 first.

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/dew' TESTARGS='-run TestAccDewCsmsSecretVersion_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dew -v -run TestAccDewCsmsSecretVersion_basic -timeout 360m -parallel 4
=== RUN   TestAccDewCsmsSecretVersion_basic
=== PAUSE TestAccDewCsmsSecretVersion_basic
=== CONT  TestAccDewCsmsSecretVersion_basic
--- PASS: TestAccDewCsmsSecretVersion_basic (57.95s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dew       58.037s
```

### CODE REVIEW

File: huaweicloud/services/dew/data_source_huaweicloud_csms_secret_version.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+			},
+			"version": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"secret_text": {
+				Type:      schema.TypeString,
+				Computed:  true,
+				Sensitive: true,
```
Comment (niuzhenguo) [inline huaweicloud/services/dew/data_source_huaweicloud_csms_secret_version.go:39]: just a question, if we set Sensitive here, can we reference it directly?

Comment (chengxiangdong) [reply huaweicloud/services/dew/data_source_huaweicloud_csms_secret_version.go:39]: yes, i have verified that the value can be used in other provider scripts

File: docs/data-sources/csms_secret_version.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to obtain the CSMS secrets.
+  If omitted, the provider-level region will be used.
+
+* `secret_name` - (Required, String) The name of the CSMS secret to query.
+
+* `version` - (Optional, String) The version ID of the CSMS secret version to query.
+  If omitted, the last version will be used.
```
Comment (ShiChangkuo) [inline docs/data-sources/csms_secret_version.md]: s/last/latest/

File: docs/data-sources/csms_secret_version.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+* `id` - The data source ID.
+
+* `secret_text` - The plaintext of a secret in text format.
+
+* `kms_key_id` - The ID of the KMS CMK used for secret encryption.
+
+* `status` - The status of the CSMS secret version.
+
+* `create_time` - Time when the CSMS secret version created, in UTC format.
```
Comment (ShiChangkuo) [inline docs/data-sources/csms_secret_version.md]: Please change it to **created_at** as we discussed before.

**Comment by niuzhenguo** [comment]
helped to update

**Pull request merged**